Cargo.toml

```diff
@@ -139,6 +154,9 @@ version = "2"
 opt-level = 3
 lto = "thin"
 incremental = true
+# uncomment the 2 lines below when building with the locktick feature
```

Comment:
Should this be in the dev mode as well in case we want to use it there?

Reply:
It can be used with any profile, though I don't think we're typically running nodes with the `dev` one (i.e. without `--release`), as they'd be rather slow; I looked into ways of enabling profile overrides when specific features are enabled, but the stable solutions are currently somewhat hacky. That's why the 2 required extra options are commented out; they can be moved under any profile when they're being enabled, though.